plugins/axios.js

```diff
@@ -0,0 +1,12 @@
+export default function ({ $axios, app, store, $nuxt }) {
+    $axios.onError((error) => {
+        if (error.response.status === 401) {
+            store.dispatch('loader/loaderState', { active: false });
+            app.$auth.logout();
+            app.$toast.info(
+                app.i18n.t('statusMessaging.authentication.logoutError'),
+            );
+        }
+        throw error;
```

Comment:
This will handle all 401 error responses while rethrowing the error so the can each be handled in different parts of the code in addition to this.